### PR TITLE
Bug 1043928 - Manually expand browser chrome

### PR DESCRIPTION
**Pull request by benfrancis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ca24f8d832fd33cca15111ed72cbdd265439aa4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b9b83766b6e7108a431c236fef3de0f526d7dc37

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ba59d541a7311e00c140eb281fec424cecfd4e49

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=32ba47154c395f2a7a0341b35b8c58cddc920578

File: apps/system/test/unit/app_chrome_test.js [hunk 1]
```diff
@@ -464,11 +464,23 @@ suite('system/AppChrome', function() {
   });
 
   suite('URLBar', function() {
-    test('click', function() {
+    test('click when collapsed', function() {
```
Comment (albertopq) [inline apps/system/test/unit/app_chrome_test.js]: Can we add tests to cover when the gobal-search-request event is triggered?

Comment (benfrancis) [inline apps/system/test/unit/app_chrome_test.js]: Thanks for getting to this so quickly Alberto! I think the global-search-request event dispatcher is covered by line 482? It's also covered by the integration test.

**Pull request merged**